lib/pipeline_controller.py

```diff
@@ -73,9 +126,99 @@ def process_all_sites_data(self, end_date, batch_num):
             if not self.ignore_cache:
                 self.s3_client.set_cache({"last_poll_date": poll_date.isoformat()})
 
-            self.logger.info(f"Finished batch {batch_num+1}: {poll_date_str}")
+            self.logger.info(f"Finished batch {batch_num+1}: {poll_date.isoformat()}")
             self.process_all_sites_data(end_date, batch_num + 1)
 
+    def process_broken_orbits(self, start_date, end_date):
+        """
+        Re-queries individual sites with unhealthy data from the past 30 days (a limit
+        set by the API) to see if any data has since been recovered
+        """
+        create_table_query = build_redshift_create_table_query(
+            self.redshift_visits_table, start_date, end_date
+        )
+        self.redshift_client.connect()
+        self.redshift_client.execute_transaction([(create_table_query, None)])
+        recoverable_site_dates = self.redshift_client.execute_query(
+            REDSHIFT_RECOVERABLE_QUERY
+        )
```

Comment:
Perhaps move this down, I had to hunt around for where this variable was set.

Reply:
Unfortunately it has to go in between the temporary table creation/deletion, so it could only really move down one line.